tests/scripts/all.sh

```diff
@@ -121,6 +158,17 @@ fi
 #
```

Comment:
I suggest if the user has specified a specific OpenSSL or GNUTLS version, we should check we can find them in the path (`hash`?) else report an error. Otherwise if we don't, what error will be provided?